### ISSUE TITLE
Tahrongi Canyon crashes are caused by charming and proceeding to "Fight" using Cactus Grub [BUG]

### ISSUE THREAD

**Issue by Soleece** [HorizonFFXI/HorizonXI-Issues#1664]
**What does this bug affect?**

Originally, I thought it may be a single Cactus Grub that was causing this, but eventually I tried with a second one and the same thing happened. I don't want to test a bug that crashes a zone continuously on an active server, so I'm going to recommend you temporarily remove all the Cactus Grubs from that zone (once you test and recognize the bug), until it is addressed. 

**Steps to Reproduce**

Charm a Cactus Grub in the SW corner of the map. 
Use the "Fight" pet command. 
and the zone instantly crashes

**Expected behavior**

The zone shouldn't be crashing. 

**Screenshots**

**Setup Information**

I'm using Version 1.1 as stated in the launcher. And I'm on Windows 11. The bug seems to be specific to that monster, since it only happens int hat zone. 

**Additional context**

I was leveling Beastmaster, so that was my main job at the time and I didn't check charming it on other jobs. If I charm a Cactus Grub, in that corner, and proceed to use fight, within range on another mob, the zone instantly crashes and refreshes.

**Comment by KamrenCodes** [comment]
I have had this issue in Tahrongi as well. Not just with grubs, but also with various other mobs.

I have also replicated this same crash in the dunes with Lizards. I am currently in Kon Highlands, seeing if it's stable or if this issue persists.

**Comment by Soleece** [comment]
Ok.
I know I was fighting those monsters the other day. I don't know if I was
actually charming the Cactus Grubs or not though.
I was able to replicate the zone crashes on every attempt though.

I sort of assumed it may have something to do with the fact that they're
era+ mobs. I did test the Canyon Wasps nearby. They were find and did not
cause any zone crashes.

I feel like I remember fighting using the Cactus Grub the other day, but
I'm not entirely sure. I could have been using the wasps. Or the zone could
have been crashing and I may not have been noticing that was the cause.

Let me know what you figure out.
